### ISSUE TITLE
Downlink on BLE links only sends the first couple of packets and discards everything else

### ISSUE THREAD

**Issue by glennvdw** [bitcraze/crazyflie-firmware#81]
When connecting to the crazyflie 2.0 not all DEBUG_PRINT(..) console messages are actually received at the client side.. only the first 20 or so data packets are sent by the crazyflie

I added in worker.c inside the infinite loop, right before xQueueReceive the following code 
```C
consolePuts("I will not reach the client..");
consoleFlush();
```
This results in an infinite amount of received strings when using a USB link but it does not work on BLE links. The buffer is never flushed and gets full I assume.. 

I used the official iOS app as a starting point for the client code.

**Comment by ataffanel** [comment]
This looks like an old bug that was since fixed, did you upgrade your Crazyflie with the latest firmware?

**Comment by glennvdw** [comment]
The nrf51 was updated using the iOS app. For the stm32 I tried lots of different versions. Currently running the latest commit on this repo.

This person seems to have reported the same problem about a month ago: https://forum.bitcraze.io/viewtopic.php?t=1700

**Comment by glennvdw** [comment]
Which version do you recall to be definitely working?

**Comment by ataffanel** [comment]
When pushing this change: https://github.com/bitcraze/crazyflie2-nrf-firmware/commit/f19059868f171449a5b18c0254401ef28a07c6b3 the downlink was working (at least I got the full console at startup). I cannot test with the iOS app right now but I am going to implement the downlink in the Android App.

**Comment by ataffanel** [comment]
I could reproduce the problem and made a ticket in the NRF firmware (the bug is in the NRF51). I am looking at it.

**Comment by ataffanel** [comment]
Fixed in bitcraze/crazyflie2-nrf-firmware#10.
